src/components/modal/v2/parts/views/ShortTerm/Content.jsx

```diff
@@ -0,0 +1,72 @@
+/** @jsx h */
+import { h, Fragment } from 'preact';
+import { useState } from 'preact/hooks';
+import Icon from '../../Icon';
+import Instructions from '../../Instructions';
+import Donut from '../../Donut';
+import ProductListLink from '../../ProductListLink';
+import styles from './styles.scss';
+
+import { useServerData } from '../../../lib/providers';
+
+export const ShortTerm = ({ instructions, linkToProductList, disclosure, donutTimestamps, contentBodyRef }) => {
+    let qualifying;
+    let periodicPayment;
+    if (useServerData()?.views?.length > 0) {
+        qualifying = useServerData().views[0].meta.qualifying;
+        periodicPayment = useServerData().views[0].meta.periodicPayment;
+    }
+
+    const [expandedState] = useState(false);
+    return (
+        <Fragment>
+            <style>{styles._getCss()}</style>
+            <div className="content__container">
+                <main className="main">
+                    <div className="content__body" ref={contentBodyRef}>
+                        <div className="dynamic__container">
+                            <div className="content__row dynamic">
+                                <div className="content__col">
+                                    <div className="content__row donuts">
+                                        <div
+                                            className={`donuts__container ${
+                                                qualifying === 'true' ? 'donuts__qualifying' : 'donuts__non_qualifying'
+                                            }`}
+                                        >
```

Comment:
```suggestion
                                            className="donuts__container">
```

I actually just got out of a 3iab with Scott and brought up that the other countries' donuts didn't have percentages inside of them and if that was intentional. A decision was made to remove the percentage from inside of the US qualifying donuts so we can simplify this class now by removing the check.